### PR TITLE
Fiber v2, code format, typo,test case, go 1.15 and more

### PR DESCRIPTION
**Pull request by BennyThink**
Five commits contain:

### Fiber
* upgrade to fiber v2

### code style and minor changes
* remove useless file and fix typo
* remove redundant variables
* remove useless exportable variables/functions
* Makefile change
* upgrade to go 1.15
* remove wrong go test comments

### test case
* add test case for converter, use deferInit to make test more simple
* complete test case with 86.3% coverage

## mirror
* go mod replace: use our own mirror now(webp and fiber)
* add banner, show version, remove fiber startup message, add server header

image

### CODE REVIEW

**Comment by BennyThink** [comment]
@n0vad3v 🤔Any ideas on this?

**Review by muzi502 — COMMENTED** [review]
/lgtm

**Comment by BennyThink** [comment]
你说的对！突然手抖

Xian Li <notifications@github.com> 于 2020年11月21日周六 12:13写道：

File: scripts/webp_server.spec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Name:    webp-server
-Version: 0.1.2
+Version: 0.2。1
```
[the remark this answers is not in the document]
Comment (BennyThink) [reply scripts/webp_server.spec]: dei 这个nova fix了，你看到的是outdated

**Review by n0vad3v — APPROVED** [review]

**Pull request merged**